executor/api/rest/utils.go

```diff
@@ -68,7 +68,7 @@ func CombineSeldonMessagesToJson(msgs []payload.SeldonPayload) (payload.SeldonPa
 	// Create JSON list of messages
 	joined := strings.Join(strData, ",")
 	jStr := "[" + joined + "]"
-	return &payload.BytesPayload{Msg: []byte(jStr)}, nil
```

Comment:
Nice one! This makes sense - we shoudl document somewhere that there is assumption that we will always return the content type of the first element (just to make sure it's explicit for users in case they don't know what the content type rules are. I assume this is basically how the java engine used to do it? If so we can merge this as it's the same as before and make sure we extend with the docs addition

Reply:
I believe that content type of what comes in every message to combiner is anyway determined by content type of initial message send by user.

After all it's the initial message independently sent to every child and output from those goes to the combiner.

This is of course under assumption thst content type cannot change in the Python Wrapper but I think we do not give option for that...